nixos/modules/config/system-path.nix

```diff
@@ -163,6 +163,7 @@ in
         "/share/kxmlgui5"
         "/share/systemd"
         "/share/thumbnailers"
+        "/share/xdg-desktop-portal"
```

Comment:
This should probably go to the portal module.

Reply:
Good point, the module actually already modifies that variable and I'm just blind.